openedx/core/djangoapps/course_date_signals/handlers.py

```diff
@@ -78,6 +80,24 @@ def _gather_graded_items(root, due):  # lint-amnesty, pylint: disable=missing-fu
     return []
 
 
+def _get_custom_pacing_children(subsection, num_weeks):
+    """
+    Return relative date items for the subsection and its children
+    """
+    items = [subsection]
+    has_non_ora_content = False
+    section_date_items = []
+    while items:
+        next_item = items.pop()
+        if next_item.category != 'openassessment':
+            has_non_ora_content = True
+            section_date_items.append((next_item.location, {'due': timedelta(weeks=num_weeks)}))
+            items.extend(next_item.get_children())
+    if has_non_ora_content:
```

Comment:
Hmmm. Just realized that this is always going to be true because the very first item you pass in (the subsection) is going to have the category of `sequential` and automatically set this to True. This might be part of the reason why the other function has the `has_score` check